independent-publisher-connectors/Proposal - GIPHY (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,878 @@
+                      "properties": {
+                        "url": {
+                          "type": "string",
+                          "description": "url"
+                        }
+                      },
+                      "description": "onclick"
+                    },
+                    "onsent": {
+                      "type": "object",
+                      "properties": {
+                        "url": {
+                          "type": "string",
+                          "description": "url"
+                        }
+                      },
+                      "description": "onsent"
+                    }
+                  },
+                  "description": "analytics"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get GIF",
+        "description": "Returns a list of GIFs related to the search string provided as input",
+        "operationId": "Get-GIF",
+        "parameters": [
+          {
+            "name": "API_KEY",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Enter your API Key.",
+            "x-ms-summary": "APIKey"
+          },
+          {
+            "name": "q",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Search String"
+            "x-ms-summary": "string"
+          },
+          {
+            "name": "limit",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "Maximum number of GIFs to be returned.",
+            "x-ms-summary": "limit"
+          },
+          {
+            "name": "offset",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "An optional results offset",
+            "x-ms-summary": "Offset"
+          },
+          {
+            "name": "rating",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Filters results by rating (g/pg/pg-13/r)",
+            "x-ms-summary": "Rating"
+          },
+          {
+            "name": "lang",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Filter results by language",
+            "x-ms-summary": "Language"
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "API Key": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "API_KEY"
+    }
+  },
+  "security": [
+    {
+      "API Key": []
+    }
+  ],
+  "tags": []
+}
+"x-ms-connector-metadata": [
+{
+  "propertyName": "Website",
+  "propertyValue": ""
+},
+{
+  "propertyName": "Privacy policy",
+  "propertyValue": ""
+},
+{
+  "propertyName": "Categories",
+  "propertyValue": ""

```

Comment:
@priyanshu870 this is not a valid category. you can choose one of the below.
Available categories are: [AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website]